dataflow/src/org/checkerframework/dataflow/cfg/DOTCFGVisualizer.java

```diff
@@ -399,4 +449,72 @@ public void shutdown() {
         }
     }
 
+	@Override
+	public void visualizeStore(S store) {
+		store.visualize(this);
+	}
+
+	@Override
+	public void visualizeThisValue(A value) {
+		this.sbStore.append("  this > " + value
+                + "\\n");
```

Comment:
I prefer having the line end with the operator. You're inconsistent in whether you end or begin with the operator.